{
  "id": "oval:org.debian:def:90304296198899563508261139273288889748",
  "class": "vulnerability",
  "metadata": {
    "title": "CVE-2016-10149 python-pysaml2",
    "affected": {
      "family": "unix",
      "platform": "Debian GNU/Linux 12",
      "product": "python-pysaml2"
    },
    "reference": [
      {
        "source": "CVE",
        "ref_id": "CVE-2016-10149",
        "ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-10149"
      }
    ],
    "description": "XML External Entity (XXE) vulnerability in PySAML2 4.4.0 and earlier allows remote attackers to read arbitrary files via a crafted SAML XML request or response.",
    "debian": {}
  },
  "criteria": {
    "operator": "AND",
    "criterias": [
      {
        "operator": "OR",
        "criterias": [
          {
            "operator": "AND",
            "criterions": [
              {
                "test_ref": "oval:org.debian.oval:tst:2",
                "comment": "all architecture"
              },
              {
                "test_ref": "oval:org.debian.oval:tst:8486",
                "comment": "python-pysaml2 DPKG is earlier than 3.0.0-5"
              }
            ]
          }
        ]
      }
    ],
    "criterions": [
      {
        "test_ref": "oval:org.debian.oval:tst:1",
        "comment": "Debian 12 is installed"
      }
    ]
  }
}
